quarkchain/core.py

```diff
@@ -263,6 +263,16 @@ def __hash__(self):
             h_list.append(getattr(self, name))
         return hash(tuple(h_list))
 
+    def to_dict(self):
+        d = dict()
+        for name, ser in self.FIELDS:
+            v = getattr(self, name)
+            if isinstance(v, Serializable):
+                d[name] = v.to_dict()
+            else:
+                d[name] = v
```

Comment:
how about deserialize bytes to hex? when `isinstance(v, FixedSizeBytesSerializer)` is true

Reply:
bytes default output is hex-like, see above example.

Reply:
but definitively, we could optimize that, but I would suggest interactive mode, which allow much richer operations.